rosidl_typesupport_cpp/package.xml

```diff
@@ -17,6 +17,8 @@
   <build_export_depend>rmw_implementation</build_export_depend>
   <build_export_depend>rosidl_generator_c</build_export_depend>
   <build_export_depend>rosidl_typesupport_c</build_export_depend>
+
+  <group_depend>rosidl_typesupport_cpp_packages</group_depend>
   <build_export_depend>rosidl_typesupport_connext_cpp</build_export_depend>
   <build_export_depend>rosidl_typesupport_introspection_cpp</build_export_depend>
   <build_export_depend>rosidl_typesupport_opensplice_cpp</build_export_depend>
```

Comment:
Can these explicit dependencies be removed in favor of the group depend ? (same in the c package)
It looks like:
- source builds: the group dependency is expanded by the buildtool and processed in order removing the need for the explicit dependency
- deb builds: we overwrite the deps in the package.xml to depend only explicitely on a specific set of typesupports (e.g [here](https://github.com/ros2-gbp/rosidl_typesupport-release/blob/12d2cbacfc7d9057bfc92976f0165e54fb7168ce/ros2/rosidl_typesupport_c/package.xml))

Reply:
I would rather no remove them at this point. For the next release we should reconsider the current patches applied in the release repo. I would propose to keep the dependencies in manifest which are also used for the packaging - without a need to apply any patches in the release repo.